routers/routes/routes.go

```diff
@@ -496,6 +496,8 @@ func RegisterRoutes(m *macaron.Macaron) {
 					})
 				})
 				m.Post("/reactions/:action", bindIgnErr(auth.ReactionForm{}), repo.ChangeIssueReaction)
+				m.Post("/deadline/update", reqRepoWriter, bindIgnErr(auth.DeadlineForm{}), repo.UpdateDeadline)
+				m.Post("/deadline/remove", reqRepoWriter, repo.RemoveDeadline)
```

Comment:
For consistency, this endpoint should be called `/delete`.

Reply:
Fixed that.